src/Wallabag/CoreBundle/Operator/PHP/Matches.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Wallabag\CoreBundle\Operator\PHP;
+
+class Matches
```

Comment:
Could you precise what these classes are used for (both `Operator\Doctrine\Matches` & `Operator\Doctrine\PHP`) ?
In the context of your PR, we can guess but when it will be inside the project, it won't be that easy to guess